packages/geo/src/lib/map/shared/controllers/geolocation.ts

```diff
@@ -396,12 +432,43 @@ export class MapGeolocationController extends MapController {
       timestamp: new Date()
     };
     this.handleFeatureCreation(position);
-    this.handleViewFromFeatures(position, zoomTo);
-    if (emitEvent) {
+    const different = this.positionsAreDifferent(
+      position,
+      this.position$.getValue()
+    );
```

Comment:
Dans quel cas de figure, la position serait identique? En théorie, si on se fit a `this.geolocation.on('change'` on ne devrait pas avoir la même position?

Reply:
possible que l'accuracy change.